addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -24,10 +24,20 @@
         </record>
 
         <record id="fp_extra" model="account.fiscal.position.template">
-            <field name="name">Régimen Extracomunitario / Canarias, Ceuta y Melilla</field>
+            <field name="name">Régimen Extracomunitario</field>
             <field name="chart_template_id" ref="account_chart_template_common"/>
         </record>
 
+        <record id="fp_not_subject_tai" model="account.fiscal.position.template">
+            <field name="name">Régimen No sujeto por reglas de localización (TAI)</field>
```

Comment:
@acysos Isn't better to add between parenthesis the usual cases as talked the other day?
```suggestion
            <field name="name">Régimen No sujeto por reglas de localización (TAI - Canarias, Ceuta, Melilla...)</field>
```